system/modules/isotope/library/Isotope/CheckoutStep/OrderConditions.php

```diff
@@ -114,7 +114,11 @@ function ($strName, &$arrDca) {
 
                 // Clone widget because otherwise we add errors to the original widget instance
                 $objClone = clone $this->objForm->getWidget($strField);
-                \Input::setPost($strField, $_SESSION['CHECKOUT_DATA'][$strField]);
+                if ($objClone instanceof \uploadable) {
+                    $_FILES[$strField] = $_SESSION['FILES'][$strField];
```

Comment:
doesn't this mean the file does not make it into checkout data as well?

Reply:
checkout data should be fine:

https://github.com/isotope/core/blob/f04cab3ab2ecb5b78f4921cdd7eeb04655beca62/system/modules/isotope/library/Isotope/CheckoutStep/OrderConditions.php#L97-L107